airbyte-integrations/connectors/source-linkedin-leads/source_linkedin_leads/manifest.yaml

```diff
@@ -0,0 +1,183 @@
+          http_method: GET
+          request_headers:
+            LinkedIn-Version: "202406"
+            X-Restli-Protocol-Version: 2.0.0
+          request_parameters:
+            q: owner
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - elements
+        partition_router:
+          type: ListPartitionRouter
+          values: '{{ config["account_ids"] }}'
+          cursor_field: account_id
+      primary_key:
+        - id
+
+    lead_form_response:
+      type: DeclarativeStream
+      name: lead_form_response
+      retriever:
+        type: SimpleRetriever
+        paginator:
+          type: DefaultPaginator
+          page_size_option:
+            type: RequestOption
+            field_name: count
+            inject_into: request_parameter
+          page_token_option:
+            type: RequestOption
+            field_name: start
+            inject_into: request_parameter
+          pagination_strategy:
+            type: OffsetIncrement
+            page_size: 1000
+            inject_on_first_request: true
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /leadFormResponses?owner=(sponsoredAccount:urn%3Ali%3AsponsoredAccount%3A{{stream_partition.account_id }})&leadType=(leadType:SPONSORED)
+          http_method: GET
+          request_headers:
+            LinkedIn-Version: "202406"
+            X-Restli-Protocol-Version: 2.0.0
+          request_parameters:
+            q: owner
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - elements
+        partition_router:
+          type: ListPartitionRouter
+          values: '{{ config["account_ids"] }}'
+          cursor_field: account_id
+      primary_key:
+        - id
+
+  base_requester:
+    type: HttpRequester
+    url_base: https://api.linkedin.com/rest
+    authenticator:
+      type: OAuthAuthenticator
+      grant_type: refresh_token
+      client_id: '{{ config["client_id"] }}'
+      client_secret: '{{ config["client_secret"] }}'
+      refresh_token: '{{ config["client_refresh_token"] }}'
+      refresh_request_body: {}
+      refresh_token_updater:
+        refresh_token_name: access_token
+        access_token_config_path:
+          - oauth_access_token
+        refresh_token_config_path:
+          - client_refresh_token
+        token_expiry_date_config_path:
+          - oauth_token_expiry_date
+      token_refresh_endpoint: https://www.linkedin.com/oauth/v2/accessToken
+
+streams:
+  - $ref: "#/definitions/streams/lead_form_response"
+  - $ref: "#/definitions/streams/lead_form"
+
+spec:
+  type: Spec
+  connection_specification:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    required:
+      - client_id
```

Comment:
If leads require new scopes compared to linkedin_ads, this would need to be a breaking change cc @FVidalCarneiro